app/views/user_mailer/password_reset.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Hi <%= @user.email %>,</p>
+
+<p>If you want to reset your password please click the link below. The reset-password-link is valid for two hours.</p>
```

Comment:
Wouldn't it be better to store time for password reset in some constant and use it across system? Especially that at some point you could decide that 2h is not enough and change it to other value.